src/rpc/blockchain.cpp

```diff
@@ -1545,7 +1545,7 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
     BuriedForkDescPushBack(softforks, "fortcanningepilogue", consensusParams.DF19FortCanningEpilogueHeight);
     BuriedForkDescPushBack(softforks, "grandcentral", consensusParams.DF20GrandCentralHeight);
     BuriedForkDescPushBack(softforks, "grandcentralepilogue", consensusParams.DF21GrandCentralEpilogueHeight);
-    BuriedForkDescPushBack(softforks, "nextnetworkupgrade", consensusParams.DF22NextHeight);
+    BuriedForkDescPushBack(softforks, "Metachain", consensusParams.DF22Metachain);
```

Comment:
Case

Reply:
Fixed in 8b51126bc0a87b835a5209b4a197ca65fc61ebc5